src/merkledag.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = new Buffer(`
```

Comment:
> we need to have a section in our main projects (js-ipfs, js-ipfs-api and js-libp2p) that explains our users how to 'webpack' all of this stuff, 

Yes, but we still should aim to make that explanation as simple as possible :)
> I don't believe that js-ipfs will ever stop needing a special webpack config.

I have concrete plans for this and a roadmap in my head on how to get there, it is coming soon. Dropping `fs.readFile` is the first step on the road there.